components/json_path.rst

```diff
@@ -81,14 +81,12 @@ the data. This method always returns an array of matching values.
 Querying with Expressions
 -------------------------
 
-The primary way to query the JSON is by passing a JSONPath expression string
-to the :method:`Symfony\\Component\\JsonPath\\JsonCrawler::find` method.
+The primary way to query the JSON is by passing a JSONPath expression string to the :method:`Symfony\\Component\\JsonPath\\JsonCrawler::find` method.
 
 Accessing a Specific Property
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Use dot-notation for object keys and square brackets for array indices. The root
-of the document is represented by ``$``::
+Use dot-notation for object keys and square brackets for array indices. The root of the document is represented by ``$``::
```

Comment:
These lines should be kept as is, the docs try to avoid to have lines more than 80 chars